### ISSUE TITLE
Add a linter

### ISSUE THREAD

**Issue by m2mathew** [reactjs-dallas/reactjs-dallas-site#18]
To get our code quality and formatting to be more consistent, let's use a linter. Hopefully, this will helps us avoid 1,000 "remove your semicolons" comments on PRs. 😆

**Comment by incorelabs** [comment]
Any updates on this? Which one are you thinking of using?

**Comment by patrickhulce** [comment]
eslint + prettier = ❤️ 

just sayin' :)

**Comment by incorelabs** [comment]
Yup, heard this combo is good. Haven't tried it though, still using the Eslint Airbnb combo.

Also, would help if there is a Editor Config file. :) Not sure how many folks use it, i like using it.

**Comment by sriverag** [comment]
Hi @m2mathew,

What do you think about adding this: https://www.gatsbyjs.com/plugins/gatsby-plugin-prettier-eslint/?
